_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+    }
+
+    .sm-col-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(10); };
+    }
+
+    .sm-col-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(11); };
+    }
+
+    .sm-col-12 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .sm-pre-1 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(1); };
+    }
+
+    .sm-pre-2 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(2); };
+    }
+
+    .sm-pre-3 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(3); };
+    }
+
+    .sm-pre-4 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(4); };
+    }
+
+    .sm-pre-5 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(5); };
+    }
+
+    .sm-pre-6 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(6); };
+    }
+
+    .sm-pre-7 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(7); };
+    }
+
+    .sm-pre-8 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(8); };
+    }
+
+    .sm-pre-9 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(9); };
+    }
+
+    .sm-pre-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(10); };
+    }
+
+    .sm-pre-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .sm-post-1 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(1); };
+    }
+
+    .sm-post-2 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(2); };
+    }
+
+    .sm-post-3 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(3); };
+    }
+
+    .sm-post-4 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(4); };
+    }
+
+    .sm-post-5 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(5); };
+    }
+
+    .sm-post-6 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(6); };
+    }
+
+    .sm-post-7 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(7); };
+    }
+
+    .sm-post-8 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(8); };
+    }
+
+    .sm-post-9 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(9); };
+    }
+
+    .sm-post-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(10); };
+    }
+
+    .sm-post-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(11); };
+    }
+
+
+    // CASE: medium screen bp
+    .md-col-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.